### DISCUSSION TITLE
PR Module Updates

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#386] [General]
@AbbyANoble and I were discussing in the meeting today the idea of how to make the PR module flow better and be more accessible for both college algebra and precalculus.

Suggestions:

- Move graphing module (should now be PR3) above long division and zeros
Make graphing module a basic overview - end behavior, multiplicities from graph and factored form
Only use factored form or easily factorable polynomials
Only real zeros

- Long Division (now PR4) see comments from Tonya #383  

- In Zero section (now PR5) - discuss rational root theorem and find zeros using that and long division
Students need to be able to find zeros of a “bigger” polynomial using some combo of rational root, or given one zero and divide, or graph to find a zero, then find the rest.
A couple at the end about complex zeros.
Also see comments from Abby #384

@tdegeorge  would you be able to look this over.

**Comment by AbbyANoble** [comment]
Adding in that the new PR3 - Graphing just needs a little tweaking to incorporate the basics of zeros (from current zero section). Then the new zero section (new PR5) about finding more difficult zeros needs writing, but is mainly for Precalculus and not an immediate need before the semester starts. Some of the current zero section can be used in the new one.

**Comment by siwelwerd** [comment]
I started fixing #383 (will be done pending a clarification from Tonya) since that is easier.  It looks like the requested improvements to PR4 are more substantial though, and would benefit from waiting until after we potentially re-organize?  In other words, I think I am suggesting the workflow:
1. Reshuffle the sections
2. Tweak "Graphs of Polynomial Functions" (Currently PR5, proposed to be PR3)
3. Rework "Zeros of Polynomial Functions (Currently PR4, proposed to be PR5) to include the notes above as well as Abby's comments in #384

I gather we are still awaiting feedback from @tdegeorge on point 1, so will hold off on the rest until then.

**Reply by siwelwerd** [reply]
1. Completed in #405
2. See #407 
3. See #408
